frontend/packages/dev-console/src/components/import/DeployImage.tsx

```diff
@@ -61,6 +61,21 @@ const DeployImage: React.FC<DeployImageProps> = ({ namespace }) => {
         privateKey: '',
       },
     },
+    serverlessRoute: {
```

Comment:
This doesn't seem to be referenced by form section.

Reply:
I'm using it now.